app/src/Sentry.ts

```diff
@@ -15,6 +15,35 @@ export const captureException = (
   });
 };
 
+export const captureFeedback = (
+  feedback: string,
+  context?: Record<string, any>,
+) => {
+  if (isSentryDisabled) {
+    return;
+  }
+
+  Sentry.captureFeedback(
+    {
+      message: feedback,
+      name: context?.name,
+      email: context?.email,
```

Comment:
i dont think this will ever not be undefined for us. so unless its required we can leave off

Reply:
oh we are asking for them in the feedback widget?